cli/chaosli/cmd/validate.go

```diff
@@ -15,20 +15,33 @@ var validateCmd = &cobra.Command{
 	Use:   "validate",
 	Short: "validate disruption config",
 	Long:  `validates the yaml of the disruption for structure.`,
-	Run: func(cmd *cobra.Command, args []string) {
+	PreRunE: func(cmd *cobra.Command, args []string) error {
 		path, _ := cmd.Flags().GetString("path")
-		fmt.Println(validate(path))
+		return validatePath(path)
 	},
+	RunE: ValidateDisruption,
 }
 
-func validate(filePath string) string {
-	if filePath == "" {
-		return pathError
+func init() {
+	validateCmd.Flags().String("path", "", "The path to the disruption file to be validated.")
+}
+
+func ValidateDisruption(cmd *cobra.Command, args []string) error {
+	path, _ := cmd.Flags().GetString("path")
+	disruption, _ := DisruptionFromFile(path)
+
+	err := disruption.Spec.Validate()
+	if err != nil {
+		return err
 	}
 
-	return "Validation TODO"
+	return nil
 }
 
-func init() {
-	validateCmd.Flags().String("path", "", "The path to the disruption file to be validated.")
+func validatePath(filePath string) error {
+	if filePath == "" {
+		return fmt.Errorf("no path given, exiting")
```

Comment:
Can you go ahead and remove `pathError` from `root.go` and make explain also call this function in its `PreRunE`?